agent/grpc-external/services/peerstream/replication.go

```diff
@@ -589,3 +609,32 @@ func makeReplicationResponse(resp *pbpeerstream.ReplicationMessage_Response) *pb
 		},
 	}
 }
+
+// nodeCheckIdentity uniquely identifies a check imported from a peering cluster
+type nodeCheckIdentity struct {
+	peer    string
```

Comment:
Adding peer name in order to avoid an odd case where two nodes from two peering clusters have the same ID with the same check ID.

Reply:
Is it possible to see that in this function, since the query includes the `peerName` to filter with?

`s.GetStore().CheckServiceNodes(nil, sn.Name, &sn.EnterpriseMeta, peerName)`

Reply:
fixed. Thanks.